app/controllers/links_controller.rb

```diff
@@ -1,5 +1,8 @@
+TRUSTED_DOMAINS = ['theforeman.org', 'redhat.com', 'orcharhino.com'].freeze
```

Comment:
cc @timogoebel @sbernhard 

are there any other domains that you'd need allowed in the redirection code (which is not hard coded in `external_url`)?

Reply:
Thoughts on making it an item in `SETTINGS`?

Reply:
WWTD - what would Tomer do? ;-)

(no idea if this needs to be a setting, as I don't anticipate it to be changed by users, but can certainly make it one if that makes handling easier?)

Reply:
I think Tomer mostly had a problem with user visible settings, but I thought about something you can override in `settings.yaml`.

Reply:
done that now